index.js

```diff
@@ -147,6 +147,13 @@ var scssContextParser = (function () {
           end : lineNumberFor(endIndex) + 1
         };
       }
+    } else {
```

Comment:
Yes, you're right about the assumption this makes. But `@keyframes` and `@media` rules are still CSS rules, so they are not a counter-example to the assumption. Currently they can't be commented at all; now they can be commented like any other CSS rule, so nothing breaks, just new opportunities are available for those who want to play with them. Nested rules are handled fine (that is, they aren't really handled at all, they just become part of the `body` / `code` -- but that's the same as nested stuff inside a mixin or whatever).

You're right that this is a rather broad rule, but I can't think of any real case where it would break. And it only ever comes into play in situations where currently your comments are just thrown away silently, so again, it doesn't break any current use case, just adds the potential for new ones.